cfn.template.yaml

```diff
@@ -218,3 +224,89 @@ Resources:
             - '*'
       Roles:
         - !Ref deliveryRole
+
+  # ALB Solution
+  LambdaPermissionForALB:
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !GetAtt Lambda.Arn
+      Action: lambda:InvokeFunction
+      Principal: elasticloadbalancing.amazonaws.com
+      SourceArn: !Ref TargetGroupForLambda
+
+  ConsentLoadBalancer:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties: 
+      IpAddressType: ipv4
+      LoadBalancerAttributes:
+        - Key: idle_timeout.timeout_seconds
+          Value: 10
```

Comment:
Is this long enough for cold starts?

Reply:
Discussed in person; not relevant in this case.